### PR TITLE
Increase OrcReader#MAX_BATCH_SIZE to 8196

### PR DESCRIPTION
**Pull request by sopel39**

### CODE REVIEW

File: lib/trino-orc/src/main/java/io/trino/orc/OrcReader.java [hunk 1]
```diff
@@ -65,7 +65,7 @@
 
 public class OrcReader
 {
-    public static final int MAX_BATCH_SIZE = 1024;
+    public static final int MAX_BATCH_SIZE = 8048;
```
Comment (raunaqmorarka) [inline lib/trino-orc/src/main/java/io/trino/orc/OrcReader.java:68]: why not 8192 ?

Comment (sopel39) [reply lib/trino-orc/src/main/java/io/trino/orc/OrcReader.java:68]: you are right :). It shouldn't affect benchmark results

Comment (Praveen2112) [reply lib/trino-orc/src/main/java/io/trino/orc/OrcReader.java:68]: Can we have any benchmark numbers post this increase ?

Comment (sopel39) [reply lib/trino-orc/src/main/java/io/trino/orc/OrcReader.java:68]: Yes. I have to rerun benchmarks due to some flakyness

**Comment by sopel39** [comment]
ORC unpart


  | label | TPCH wall time | TPC-DS wall time | TPCH CPU time | TPC-DS CPU time | TPCH peak mem | TPC-DS peak mem
-- | -- | -- | -- | -- | -- | -- | --
baseline | 750.309333 | 1794.496333 | 99157.1 | 238948.688000 | 2.063479e+09 | 1.127753e+09
orc 8k unpart | 737.433500 | 1746.449500 | 95702.4 | 229404.482167 | 2.078884e+09 | 1.123496e+09

ORC part



label | TPCH wall time | TPC-DS wall time | TPCH CPU time | TPC-DS CPU time | TPCH peak mem | TPC-DS peak mem
-- | -- | -- | -- | -- | -- | --
baseine | 833.860667 | 1143.438667 | 111812.3 | 133595.615000 | 2.127303e+09 | 1.246159e+09
orc 8k part | 815.746833 | 1106.233500 | 110167.1 | 127751.651667 | 2.144631e+09 | 1.246598e+09

**Review by kokosing — APPROVED** [review]

**Pull request merged**